_includes/version-warning.html

```diff
@@ -7,6 +7,15 @@
     <a href="{{latest_url}}">latest version</a>.
     </p>
   </div>
+{% elsif page.version == 'v.0.10.0-beta.1' %}
```

Comment:
looks like you've got an extraneous `.` after `v`

Reply:
lolyep of course you know when I saw it ...